script/QA/QA_09.txt

```diff
@@ -177,38 +177,38 @@ It's not as long as the lecture on the vampire hierarchy.
 [sha:8add83eaa646cc6d2b97ae57faf22914c3402a88]{
 -- Page 197, Offset 43125.
 -- 基本的には頼れる先輩と迷える後輩の、
-In our case, it was simply a pure, platonic conversation
+In our case, it was simply a pure, platonic%{n}conversation
 }
 [sha:4d9019bb07c8396d980d7d9198cfa4ae139a532a]{
 -- Page 197, Offset 43126.
 -- 清らかな男女交際トークです。
-between a lost junior and his reliable senior.
+ between a lost junior%{n}and his reliable senior.
 }
 [sha:ceb4ede87e45ce551a0cbf9f4b4002f991df605d]{
 -- Page 198, Offset 43127.
 -- 学校ならアタシも行ったし。
-If you're talking about school, I was there too.
+If you're talking about school,%{n}I was there too.
```

Comment:
This is a J->E antipattern. May as well translate it as something more appropriate like just:  "I went to school too"